WalletWasabi.Gui/Program.cs

```diff
@@ -151,6 +153,12 @@ private static async Task TerminateApplicationAsync()
 			var mainViewModel = MainWindowViewModel.Instance;
 			if (mainViewModel is { })
 			{
+				var window = ((IClassicDesktopStyleApplicationLifetime)Application.Current.ApplicationLifetime)?.MainWindow as MainWindow;
+				if (window is { })
```

Comment:
Can it happen that window is `null`?

Reply:
According to this: 

https://github.com/molnard/WalletWasabi/blob/e93588ce0839d90211abc8cdf802c573e92f9d62/WalletWasabi.Gui/App.xaml.cs#L28

The ViewModel created first, then the view. So theoretically it is possible to have the ViewModel but the View is not created due to some error. I am not sure it is Avalonia's internal code - better be safe.

Reply:
I agree. It's just when the check proves to be superfluous then it is confusing, so it's good to know what can and what cannot be null. But yeah, ok.